src/index.html

```diff
@@ -12,6 +12,138 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+  >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surame:</label>
+          <input type="text" name="surname"
+          id="surname" autocomplete="off"
+          required
+        >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
            required
          >
```
Якщо атрибутів більше 3, то кожен атрибут має бути на новому рядку